app/models/user.rb

```diff
@@ -44,6 +44,7 @@ class User < ApplicationRecord
            source: :service_provider_record
   has_many :sign_in_restrictions, dependent: :destroy
   has_many :in_person_enrollments, dependent: :destroy
+  has_one :fraud_event, dependent: :destroy
```

Comment:
i think there could be multiple of these?

Reply:
I spent a bit thinking this through earlier and was unsure. I didn't see an obvious way for a user to get here repeatedly. You should be blocked from doing much, right?

What I didn't consider until just now is reproofing. It seems like a user could be flagged for fraud, get approved, but then later reproof and get flagged again?

I initially had this as a `.last` which felt kind of inarticulate. I guess an alternative is to delete this row when the rake task is run for them. Probably better to just go back to `has_many` and `.last`.

Reply:
yeah safer to assume multiple and do `.last` IMO